site/src/modules/dashboard/Navbar/NavbarView.tsx

```diff
@@ -1,239 +1,134 @@
-import Drawer from "@mui/material/Drawer";
-import IconButton from "@mui/material/IconButton";
 import type * as TypesGen from "api/typesGenerated";
 import { ExternalImage } from "components/ExternalImage/ExternalImage";
 import { CoderIcon } from "components/Icons/CoderIcon";
 import type { ProxyContextValue } from "contexts/ProxyContext";
-import { type FC, useState } from "react";
+import type { FC } from "react";
 import { NavLink, useLocation } from "react-router-dom";
-import { navHeight } from "theme/constants";
+import { cn } from "utils/cn";
 import { DeploymentDropdown } from "./DeploymentDropdown";
+import { MobileMenu } from "./MobileMenu";
 import { ProxyMenu } from "./ProxyMenu";
 import { UserDropdown } from "./UserDropdown/UserDropdown";
 
 export interface NavbarViewProps {
 	logo_url?: string;
 	user?: TypesGen.User;
+	docsHref: string;
 	buildInfo?: TypesGen.BuildInfoResponse;
 	supportLinks?: readonly TypesGen.LinkConfig[];
 	onSignOut: () => void;
 	canViewDeployment: boolean;
 	canViewOrganizations: boolean;
-	canViewAllUsers: boolean;
 	canViewAuditLog: boolean;
 	canViewHealth: boolean;
 	proxyContextValue?: ProxyContextValue;
 }
 
-export const Language = {
-	workspaces: "Workspaces",
-	templates: "Templates",
-	users: "Users",
-	audit: "Audit Logs",
-	deployment: "Deployment",
-};
-
-interface NavItemsProps {
-	className?: string;
-}
-
-const NavItems: FC<NavItemsProps> = ({ className }) => {
-	const location = useLocation();
-	const theme = useTheme();
-
-	return (
-		<nav className={className}>
-			<NavLink
-				css={[
-					styles.link,
-					location.pathname.startsWith("/@") && {
-						color: theme.palette.text.primary,
-						fontWeight: 500,
-					},
-				]}
-				to="/workspaces"
-			>
-				{Language.workspaces}
-			</NavLink>
-			<NavLink css={styles.link} to="/templates">
-				{Language.templates}
-			</NavLink>
-		</nav>
-	);
+const linkStyles = {
+	default:
+		"text-sm font-medium text-content-secondary no-underline block h-full px-2 flex items-center hover:text-content-primary transition-colors",
+	active: "text-content-primary",
 };
 
 export const NavbarView: FC<NavbarViewProps> = ({
 	user,
 	logo_url,
+	docsHref,
 	buildInfo,
 	supportLinks,
 	onSignOut,
 	canViewDeployment,
 	canViewOrganizations,
-	canViewAllUsers,
 	canViewHealth,
 	canViewAuditLog,
 	proxyContextValue,
 }) => {
-	const theme = useTheme();
-	const [isDrawerOpen, setIsDrawerOpen] = useState(false);
-
 	return (
-		<nav
-			css={{
-				height: navHeight,
-				backgroundColor: theme.palette.background.paper,
-				borderBottom: `1px solid ${theme.palette.divider}`,
-			}}
-		>
-			<div css={styles.wrapper}>
-				<IconButton
-					aria-label="Open menu"
-					css={styles.mobileMenuButton}
-					onClick={() => {
-						setIsDrawerOpen(true);
-					}}
-					size="large"
-				>
-					<MenuIcon />
-				</IconButton>
+		<div className="border-0 border-b border-solid h-[72px] flex items-center leading-none px-6">
```

Comment:
They shouldn’t, but since we’re not using Tailwind’s preflight and are still relying on MUI’s reset, the border doesn’t apply correctly without those classes.